tests/test_lag_2.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+import time
+import logging
+from ptf_runner import ptf_runner
+from platform_fixtures import conn_graph_facts
+
+@pytest.fixture(scope="module")
+def common_setup_teardown(duthost, ptfhost, conn_graph_facts):
+    # TODO: get the testbed_type
+
+    lag_facts = duthost.lag_facts(host = duthost.hostname)['ansible_facts']
+    if lag_facts['names'] == []:
+        pytest.skip("No lag configuration found in %s" % duthost.hostname)
+
+    test_minlink = True
+    test_rate = True
+
+    # Add/update the public key
+    # TODO: convert add_container_to_inventory.yml
+
+    mg_facts = duthost.minigraph_facts(host=hostname)['ansible_facts']
+    fanout_neighbors = conn_graph_facts['device_conn']
+    vm_neighbors = mg_facts['minigraph_neighbors']
+
+    # Copy PTF test into PTF-docker for test LACP DU
+    test_files = ['lag_test.py', 'acs_base_test.py', 'router_utils.py']
+    for test_file in test_files:
+        src = "roles/test/files/acstests/%s" % test_file
+        dst = "/tmp/%s" % test_file
+        ptfhost.copy(src=src, dest=dst)
+    
+    # Copy tests to the PTF-docker
+    ptfhost.copy(src="ptftests", dest="/root")
+
+    # Inlucde testbed topology configuration
+    if testbed_type == 't1-lag':
+        # TODO
```

Comment:
Looks like include_vars is a ansible embedded method too.
I will check the api document later